pynest/nest/tests/test_quantal_stp_synapse.py

```diff
@@ -101,15 +97,16 @@ def test_QuantalSTPSynapse(self):
         vm_reference = numpy.array(nest.GetStatus(
             [voltmeter[0]], 'events')[0]['V_m'])
 
-        vm.shape = (n_trials, t_tot)
-        vm_reference.shape = (n_trials, t_tot)
+        vm.shape = (n_trials, int(t_tot))
+        vm_reference.shape = (n_trials, int(t_tot))
 
         vm_mean = numpy.array([numpy.mean(vm[:, i])
                                for i in range(int(t_tot))])
```

Comment:
I think you can avoid the list comprehension here, NumPy supports row- and column-wise means.